iac-extensions/kubernetes/src/main/java/org/sonar/iac/helm/LineNumberCommentInserter.java

```diff
@@ -41,8 +41,11 @@ public static String addLineComments(String content) {
     var lastIndex = 0;
     while (matcher.find()) {
       lineCounter++;
-      sb.append(matcher.group("lineContent"));
-      sb.append(commentLineNumber(lineCounter));
+      var lineContent = matcher.group("lineContent");
+      sb.append(lineContent);
+      if (!"---".equals(lineContent)) {
```

Comment:
YAML also has `...` for end of document. I've never seen it being used, but maybe we should also add it here?

Reply:
Good idea, added